### ISSUE TITLE
W8709-13PC.Lopes.2018 no temperature

### ISSUE THREAD

**Issue by chrisdane** [nickmckay/LiPDverse#134]
Hi

I think https://lipdverse.org/globalHolocene/1_0_0/W8709-13PC.Lopes.2018.html is no temperature?

Kind regards,
Chris

**Comment by nickmckay** [comment]
Yep, that looks like a glitch. We'll fix that in the next version of this file.
